go.sum

```diff
@@ -1,9 +1,19 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
+cloud.google.com/go v0.31.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
+cloud.google.com/go v0.34.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
+cloud.google.com/go v0.36.0 h1:+aCSj7tOo2LODWVEuZDZeGCckdt6MlSF+X/rB3wUiS8=
+cloud.google.com/go v0.36.0/go.mod h1:RUoy9p/M4ge0HzT8L+SDZ8jg+Q6fth0CiBuhFJpSV40=
```

Comment:
There are a lot of duplicates in here (fingerprints for multiple versions of the same module). I am not sure when/why the `go` tool chooses to do this. But, for what it's worth, if I remove the `go.sum` file and then re-generate it via `GO111MODULE=on go test ./...`, it results in a file with some of these extra version lines gone:
https://github.com/fullstorydev/hauser/compare/jh/skinnier-go-sum